Lib/test/test_typing.py

```diff
@@ -7126,6 +7126,9 @@ class TD(TypedDict):
         self.assertIs(type(a), dict)
         self.assertEqual(a, {'a': 1})
 
+    def test_orig_bases(self):
+        self.assertEqual(ChildTotalMovie.__orig_bases__, (ParentNontotalMovie,))
```

Comment:
Could you add some more tests? Suggestions: a TypedDict with no bases, one with multiple bases, a generic TypedDict.

Reply:
`__orig_bases__` on generic TypedDicts are already tested quite extensively, e.g.

https://github.com/python/cpython/blob/4037df12250d4972651793f3d224da8e23ebe271/Lib/test/test_typing.py#L7017

No bases and multiple bases sound worth adding, though

Reply:
I added quite a few :)

Reply:
Should I remove the checks on generic TypedDicts then? Or move them all here?

Reply:
> Should I remove the checks on generic TypedDicts then? Or move them all here?

I think it's probably fine as is, actually. A little bit of duplication in tests isn't a massive issue, and it makes sense to both:

1). Test `__orig_bases__` in the test method for testing generic TypedDicts
2). Test generic TypedDicts in the test method for testing `__orig_bases__`